src/main/java/sign_up/use_case/DsGateway.java

```diff
@@ -0,0 +1,17 @@
+package sign_up.use_case;
```

Comment:
Rename to `SignUpDsGateway` so its obvious that its the DsGateway specific to this use case